### ISSUE TITLE
(under TF 0.7) it looks for saved files in save/save/

### ISSUE THREAD

**Issue by danbri** [hardmaru/write-rnn-tensorflow#4]
I had to copy the contents of save/ into save/save/ to get this to run. This is on OSX under TF 0.7, haven't tried it on earlier versions of TF.

**Comment by danbri** [comment]
This was running from top level of the repo,

    python sample.py --filename example_name --sample_length 1000

**Comment by hardmaru** [comment]
Yeah, TF changed how it handled checkpoint paths somewhere between 0.6 to 0.7.  It's rather annoying, affecting my other projects too sometimes.

**Comment by Arn-O** [comment]
Can you try it with this checkpoint file:

https://github.com/Arn-O/write-rnn-tensorflow/blob/python3/save/model.ckpt-0

Just drop in the `save` folder.

**Comment by hardmaru** [comment]
Thanks! feel free to submit the pull request to put the updated model in too-  The old model was trained on one of the very first open source versions of TF ...

**Comment by Arn-O** [comment]
On it!
